.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,12 @@
-Fixes #
-
 <!--
-Please follow the naming conventions in the "guidelines for contributing" link above or the pull request may be rejected.
+Naming convention for Pull Requests:
```

Comment:
We already have this documented somewhere; do not document it again. While it is good to have it here, duplicating documentations add to maintenance burden and is prone to "forgetting to update one of the duplicates".